### PR TITLE
Update draft-tigress-requirements.md

### PR DESCRIPTION
**Pull request by dimmyvi**

### CODE REVIEW

**Comment by alexpell00** [comment]
It looks like the build checks passed, does that mean the format is good?

**Comment by dimmyvi** [comment]
Most Probably.
There still possible issues when we start converting it to xml - as a part of submission to data tracker, but we’ll handle it at that time, 

Dmitry Vinokurov
 Pay | 669-269-5514 | Infinite Loop 6, Cupertino, CA | ***@***.***

File: draft-tigress-requirements.md [hunk 1]
```diff
@@ -73,6 +73,79 @@ TODO Introduction
...
+- Sender (device) - a device initiating a transfer of Provisioning Information to a Receiver that can provision this credential.
+- Receiver (device) - a device that receives Provisioning Information and uses it to provision a new credential.
+- Intermediary (server) - an intermediary server that facilitates transfer of provisioning information between Sender and Receiver.
+
+# Use Cases
+
+## 1. Let's say Ben owns a vehicle that supports digital keys which comply with the CCC open standard. Ben would like to let Ryan borrow the car for the weekend. Ryan and Ben are using two different mobile phones with different operating systems. In order for Ben to share his car key to Ryan for a weekend, he must transfer some data to the receiver device. The data structure shared between the two participants is defined in the CCC. In addition, the CCC requires the receiver to generate required key material and return it to the sender to sign and return back to the receiver. At this point, the receiver now has a token that will allow them to provision their new key with the car.
```
Comment (tfpauly) [inline draft-tigress-requirements.md:88]: You don't want to put a `##` here. The formatting looks very weird, see https://dimmyvi.github.io/tigress-requirements/Add-use-cases-and-terminology/draft-tigress-requirements.html#name-1-lets-say-ben-owns-a-vehic

Comment (dimmyvi) [reply draft-tigress-requirements.md:88]: Thanks for catching this, I should've looked at the preview generating. Removing numbers

Comment (tfpauly) [reply draft-tigress-requirements.md]: @dimmyvi no, the problem isn't the numbers, it's the `##`. That marks these lines as headers.

Comment (dimmyvi) [reply draft-tigress-requirements.md]: fixed

**Pull request merged**